lib/recommender_system.py

```diff
@@ -1,25 +1,47 @@
 #!/usr/bin/env python
 """
-This is a module that contains the main class and functionalities of
-the recommender systems.
+This is a module that contains the main class and functionalities of the recommender systems.
 """
+import numpy
+from content_based import ContentBased
+from util.data_parser import DataParser
+from util.recommender_configuer import RecommenderConfiguration
 
 
 class RecommenderSystem(object):
     """
-    A class that will combine the content-based and user-based method,
+    A class that will combine the content-based and collaborative-filtering,
     in order to provide the main functionalities of recommendations.
     """
-    def __init__(self, data):
+    def __init__(self, n_factors=4):
         """
         Constructor of the RecommenderSystem.
         """
-        pass
+        # DataParser.process()
+        ratings = numpy.matrix(DataParser.get_ratings_matrix())
+        self.config = RecommenderConfiguration()
+        if self.config.get_content_based() == 'LDA':
+            pass
+        if self.config.get_content_based() == 'LDA2Vec':
```

Comment:
If there is an error in the configuration file this won't run. if the input is LD for example. let's use a default method or simply quit if there is a mistake in the input